ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -207,7 +213,7 @@ public BlobStore(ReplicaId replicaId, StoreConfig config, ScheduledExecutorServi
     errorCount = new AtomicInteger(0);
     currentState = ReplicaState.OFFLINE;
     remoteTokenTracker = replicaId == null ? null : new RemoteTokenTracker(replicaId);
-    logger.debug(
+    logger.info(
```

Comment:
Double check if it only print once during the bootup time.

Reply:
printed once per partition but only during server startup